Tests/js/views/fields/assets/ez-selection-editview-tests.js

```diff
@@ -575,9 +575,101 @@ YUI.add('ez-selection-editview-tests', function (Y) {
         },
     });
 
+    regressionTest = new Y.Test.Case({
+        name: "eZ Selection View test for UI behavior",
+
+        _getFieldDefinition: function (required, multiple, options) {
+            return {
+                isRequired: required,
+                fieldSettings: {
+                    isMultiple: multiple,
+                    options: options,
+                }
+            };
+        },
+
+        _getField: function (fieldValues) {
+            return {
+                fieldValue: fieldValues,
+            };
+        },
+
+        setUp: function () {
+            this.field = {};
+            this.jsonContent = {};
+            this.jsonContentType = {};
+            this.jsonVersion = {};
+            this.content = new Y.Mock();
+            this.version = new Y.Mock();
+            this.contentType = new Y.Mock();
+            Y.Mock.expect(this.content, {
+                method: 'toJSON',
+                returns: this.jsonContent
+            });
+            Y.Mock.expect(this.version, {
+                method: 'toJSON',
+                returns: this.jsonVersion
+            });
+            Y.Mock.expect(this.contentType, {
+                method: 'toJSON',
+                returns: this.jsonContentType
+            });
+
+            this.view = new Y.eZ.SelectionEditView({
+                container: '.container',
+                field: this.field,
+                content: this.content,
+                version: this.version,
+                contentType: this.contentType
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "EZP-24716: Should validate after adding a selection": function () {
+            var container = this.view.get('container'),
+                option,
+                that = this;
+
+            this.view.set('fieldDefinition', this._getFieldDefinition(true, false, options));
+            this.view.render();
+            this.view.set('active', true);
+            this.view.set('showSelectionUI', true);
+
+            this.view.validate();
+            Y.Assert.isTrue(!!this.view.get('errorStatus'), "An error should be detected because field is requiered and empty");
+
+            option = container.one('.ez-selection-options li');
+            option.simulateGesture('tap', function () {
```

Comment:
If I understand the fix, it should call validate on changing `values` attrib (to be precise on add or remove). So I guess simulating user gestures is not needed here as you should just set the value for `values` attrib, then set it again to changed value and just check if the field was validated properly. Please verify it because I may not understand the mechanism.